iconservice/iconscore/icx.py

```diff
@@ -74,7 +74,9 @@ def get_balance(self, address: 'Address') -> int:
     # noinspection PyBroadException
     def _is_icx_send_defective(self) -> bool:
         try:
-            governance_score = IconScoreContextUtil.get_icon_score(self._context, GOVERNANCE_SCORE_ADDRESS)
+            governance_score = IconScoreContextUtil.get_builtin_score(
+                self._context, GOVERNANCE_SCORE_ADDRESS)
+
             if governance_score is not None:
```

Comment:
already check it.
remove duplicate logic

Reply:
Remove "if governance_score is not None"